draft-ietf-intarea-provisioning-domains.xml

```diff
@@ -324,11 +324,11 @@
 
             <t hangText="RA message header : ">(16 octets) When the R-flag is
             set, a full Router Advertisement message header as specified in
-            <xref target="RFC4861"/>. The 'Type', 'Code' and 'Checksum' fields
-            (i.e. the first 32 bits), MUST be set to zero by the sender and
-            ignored by the receiver. The other fields are to be set and parsed
-            as specified in <xref target="RFC4861"/> or any updating
-            documents.</t>
+            <xref target="RFC4861"/>. The 'Type' MUST be set to 134, the value
+            for "Router Advertisement". The 'Code' MUST be 0. The 'Checksum' MUST
```

Comment:
I find 'The Code MUST be 0' a bit ambiguous. Is it for the receiver or the sender ?

Reply:
I agree with Pierre on this one.

Reply:
I was capturing the comment from Erik Kline on this one; what do we want to say? I think it should be that we MUST set it to 0 when sending, and MUST ignore it when receiving?

Reply:
Indeed, the good old behavior of being strict in what you send and lenient at what you accept. Go for it !

Reply:
Sounds good! Updated!